src/index.js

```diff
@@ -66,13 +56,15 @@ const iSamples_componentPack = {
   searchFields: {
     ...defaultComponentPack.searchFields,
     text: TextSearch,
+    container: SearchFieldContainer,
     "date-range-facet": iSamples_RangeFacet,
   }
 }
 
 // Construct the solr client api class
 const solrClient = new SolrClient({
   url: config.solr_url,
+  // only use the fields with hidden === false
```

Comment:
Is this comment still true?  I thought that wasn't the case.

Reply:
Sorry, I should remove it.